### DISCUSSION TITLE
Valkey 7.2.6 support for JSON

### DISCUSSION THREAD

**Discussion by anurag-tally** [valkey-io/valkey#1035] [Q&A]
Hello Community - Quick check, does Valkey 7.2.6 supports JSON like redisJSON? I tried installing redis OSS 7.2.4, no luck, I think its because redis 7.2.4 == valkey 7.2.6

**Comment by madolson** [comment]
Neither Redis 7.2.4 nor Valkey 7.2.6 have JSON natively. redisJSON has always been a separate module and needs be installed separately.

**Reply by anurag-tally** [reply]
Thanks a lot @madolson
